packages/rocketchat-file-upload/server/config/FileSystem.js

```diff
@@ -17,7 +17,12 @@ const FileSystemUploads = new FileUploadClass({
 
 			if (stat && stat.isFile()) {
 				file = FileUpload.addExtensionTo(file);
-				res.setHeader('Content-Disposition', `attachment; filename*=UTF-8''${ encodeURIComponent(file.name) }`);
+				if (req.query && req.query.hasOwnProperty('disposition')) {
```

Comment:
Why not?
```javascript
const disposition = req.query && req.query.hasOwnProperty('disposition') ? req.query.disposition : 'attachment';
if (disposition) {
	res.removeHeader('Content-Security-Policy');
}
res.setHeader('Content-Disposition', `${ req.query.disposition }; filename*=UTF-8''${ encodeURIComponent(file.name) }`);
```

And why need to remove the `Content-Security-Policy`?

Reply:
> And why need to remove the Content-Security-Policy?

because i get error:

![_59](https://user-images.githubusercontent.com/13063572/53789394-f3a07180-3f2c-11e9-9630-634f07991b1f.png)

and probably rude fix:

```js
if (disposition === 'inline') {
	res.removeHeader('Content-Security-Policy');
}
```